### PR TITLE
Removed Modified Files By TayMcKenzieNZ

### PR DESCRIPTION
**Pull request by TayMcKenzieNZ**
This pull request keeps the script from doing as it is intended to do and still works with North Yankton, however removes 3 years of Tay's work. I have been nice enough to provide you modified files which open the mansion gates for access, deleted and removed the underwater gate, removed the collisions at the beach party DJ booth, as well as around the island that Rockstar added, to stop boats from docking and the player being able to climb onto land.

I have also credit __**ALL**__ people involved in this project, respectfully!

Please take this as a lesson that you should ideally credit the original creator, or remove their content when asked.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -176,72 +94,10 @@ I don't use any fancy frameworks or any other Cayo Perico related resources, so
...
-
-You are not allowed to sell this nor re-distribute it. 
-
-You are not allowed to change/add a license. If you want to modify or make an agreement, you can contact me. 
-
-Pull requests are accepted as long as they do not contain breaking changes. 
+# CREDITS
 
-You can read more here [HERE](https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license)
+Thank you to [TayMcKenzieNZ](https://github.com/taymckenzienz) for bringing Cayo Perico to FiveM.
```
Comment (alberttheprince) [inline readme.md:105]: This is unnecessary, as there is a link to the original repo in the main intro of the post.

File: readme.md [hunk 2]
```diff
@@ -176,72 +94,10 @@ I don't use any fancy frameworks or any other Cayo Perico related resources, so
...
 
+Thank you to [AvaN0x](https://github.com/AvaN0x) for fixing the minimap so Cayo Perico appears in the pause menu when pressing ESC on the keyboard.
```
Comment (alberttheprince) [inline readme.md:107]: This is a proper credit, as Avan0x does not appear in the credits, please follow this format.

File: readme.md [hunk 3]
```diff
@@ -152,13 +76,7 @@ Set it to false and request your IPLs the correct way.
...
 `Original Hangar Collision` folder provided. If you happen to have the '"[Cayo Perico Shops MLO](https://forum.cfx.re/t/mlo-cayo-perico-shops/1994908)' resource, as well as mine, the server refuses to read the collision file for shops, but instead, read mine, making the shop collisions not work. 
 
-Just copy and paste mine, or the Cayo Perico Shops collision file **(h4_islandairstrip_12)** into the ybn folder. 
-
----------------------------------------
-
-# Frameworks & Other Cayo Perico Resources / Scripts Support:
-
-I don't use any fancy frameworks or any other Cayo Perico related resources, so you'll have to troubleshoot it on your **own.** These are simply ymap, ytyp and ybn files.
+Just copy and paste the one provided, or the Cayo Perico Shops collision file **(h4_islandairstrip_12)** into the ybn folder.
```
Comment (alberttheprince) [inline readme.md:85]: Please do not delete flavour text/commentary by the original creator.

File: readme.md [hunk 4]
```diff
@@ -49,85 +35,23 @@ I hope you enjoy 😊
...
+-----------------
 # CONTENTS 🏝️
 
 - Spawns Cayo Perico alongside Los Santos!
 
-- Working lighthouse
-
 - **Heist props**  - Can be changed in the `entitysets.lua` file within the **scripts** folder
 
 - Removed snow falling from North Yankton 🌨️
```
Comment (alberttheprince) [inline readme.md:54]: Please do not delete features list, this is important for knowing what features and general GTA props were added to the map. Additionally the flavor text was part of the original repo.

File: readme.md [hunk 5]
```diff
@@ -49,85 +35,23 @@ I hope you enjoy 😊
...
+> can't figure it out, open an issue on their resource. 
 
 ---------------------------------------
 
 # Configuring PMMS 
 
 **Please see [Kibook's PMMS](https://github.com/kibook/pmms) for installation instructions and troubleshooting**
 
-------------------
-
```
Comment (alberttheprince) [inline readme.md:63]: Please do not remove info on static emitters, something that is important to the functionality of this resource.

File: readme.md [hunk 6]
```diff
@@ -49,85 +35,23 @@ I hope you enjoy 😊
 >
 > you will need to read pmms installation 
 > instructions on the GitHub repository, and if you
-> can't figure it out, open an issue. 
->
-> I'm **not** here to hold your hand.
+> can't figure it out, open an issue on their resource.
```
Comment (alberttheprince) [inline readme.md:38]: Please use proper grammar/English. Please use appropriate pronouns.

File: readme.md [hunk 7]
```diff
@@ -1,44 +1,30 @@
...
+CayoTwoIslands was a passion project of Tay's that began in December of 2020 to self teach himself Codewalker. I have modified it to work with North Yankton after discussing with TayMcKenzieNZ, who has granted me permission, so long as his modified files are not included in this fork.
 
 ---------------------------------------
 
 # **REQUIREMENTS** ⚙️
 
 - **Latest recommended artifacts** [for Windows](https://runtime.fivem.net/artifacts/fivem/build_server_windows/master/) **or** [for Linux](https://runtime.fivem.net/artifacts/fivem/build_proot_linux/master/)
 
--  [Gamebuild 2189 or higher](https://forum.cfx.re/t/tutorial-forcing-gamebuild-to-casino-cayo-perico-or-tuners-update/4784977)
+-  [Gamebuild 2189 or higher](https://docs.fivem.net/docs/server-manual/server-commands/#sv_enforcegamebuild-build)
```
Comment (alberttheprince) [inline readme.md:27]: Please do not change links to publicaly available threads.

File: readme.md [hunk 8]
```diff
@@ -1,44 +1,30 @@
...
-
 It is to work alongside [[pmms](https://github.com/kibook/pmms)] for playing content on the TV screens, as well as [[fivem-scenarios](https://github.com/kibook/fivem-scenarios)] for interacting with scenarios around the world, (like seats and scenarios) including Cayo Perico.
 
-This is a passion project of mine that began in December of 2020 to self teach myself Codewalker and has been constantly updated and developed since.
-
-Screenshots are provided in their respective folder.
-
-
-I hope you enjoy 😊
+CayoTwoIslands was a passion project of Tay's that began in December of 2020 to self teach himself Codewalker. I have modified it to work with North Yankton after discussing with TayMcKenzieNZ, who has granted me permission, so long as his modified files are not included in this fork.
```
Comment (alberttheprince) [inline readme.md:19]: Please do not add superfluous statements, this is intended to preserve the resource as published by Tay Mackenzie and contributors.

File: readme.md [hunk 9]
```diff
@@ -1,44 +1,30 @@
...
 
-Designed for RP and exploration in mind, CayoTwoIslands does everything [Cayo Improvements](https://github.com/TayMcKenzieNZ/CayoImprovements) does, except one important and highly requested feature; no more concealment.
-
-<p align="center">
-    <a href="https://github.com/TayMcKenzieNZ/CayoTwoIslands/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-GPL--3.0-blue.svg" alt="License"></a>
-    <a href="https://github.com/TayMcKenzieNZ/CayoTwoIslands/stargazers"><img src="https://img.shields.io/github/stars/TayMcKenzieNZ/CayoTwoIslands.svg?style=social&label=Stars"></a>
-    <a href="https://github.com/TayMcKenzieNZ/CayoTwoIslands/network/members"><img src="https://img.shields.io/github/forks/TayMcKenzieNZ/CayoTwoIslands.svg?style=social&label=Forks"></a>
-</p>
-
+Designed for RP and exploration in mind.
```
Comment (alberttheprince) [inline readme.md:14]: Please do not remove links, as this potentially scrubs important archival information for FiveM documentation/history.

File: readme.md [hunk 10]
```diff
@@ -1,44 +1,30 @@
...
+A slimmed-down version of Cayo intended to work with [North Yankton Enabled](https://github.com/alberttheprince/NorthYankton/). Files used with North Yankton have been restored, and if you'd like to be able to use North Yankton with Cayo, then choose this. If you have no interest in using North Yankton, please use [Tay's Original Upload](https://www.buymeacoffee.com/taymckenzienz/e/235802). This resource would not be possible without the tireless work and effort her has contributed to the community.
+
+
+I have removed some things I felt were unnecessary (personal opinion stylistically) and will make additional changes so things like ped traffic don't interfere with North Yankton's traffic and vice versa.
```
Comment (alberttheprince) [inline readme.md:6]: Please do not edit this portion, as it is my statement/preamble to the rest of the readme in light of Tay Mackenzie's original readme.

**Review by alberttheprince — CHANGES_REQUESTED** [review]
When creating PRs, please make small, targeted PRs. The changes are too large to realistically check differences in a timely manner.

Additionally please remove any references to TheIndra as they are not a contributor to this resource.

**Comment by TayMcKenzieNZ** [comment]
I don't think you understand. This was MY resource. Since you're too lazy to remove my files, I did it for you. The scritp to load cayo perico and los santos side by side, works. THAT was your intentions. AvaN0x did in fact help me with this resource; I should know as I did the resource in the first place!

File: readme.md (hunk 9 above)
Comment (TayMcKenzieNZ) [reply readme.md:14]: link is dead. isn't needed anymore.

File: readme.md (hunk 8 above)
Comment (TayMcKenzieNZ) [reply readme.md:19]: excuse me?? you are seriously starting to piss me off. I created this resource, you haven't done shit but piss me off.

File: readme.md (hunk 1 above)
Comment (TayMcKenzieNZ) [reply readme.md:105]: In case you didn't realise, the link doesn't work.

File: readme.md (hunk 2 above)
Comment (TayMcKenzieNZ) [reply readme.md:107]: If you read the original commits you would see that AvaN0x helped me fixed the minimap of cayo perico appearing 🙄. Credit the correct people, dude.

File: readme.md (hunk 3 above)
Comment (TayMcKenzieNZ) [reply readme.md:85]: I am the original creator!

File: readme.md (hunk 4 above)
Comment (TayMcKenzieNZ) [reply readme.md:54]: These are featured I, Tay McKenzie worked on. You did not do ANTYTHING. REMOVE MY FILES!!

File: readme.md (hunk 5 above)
Comment (TayMcKenzieNZ) [reply readme.md:63]: No they aren't. Learn to code properly, i did this work, not you.

File: readme.md (hunk 6 above)
Comment (TayMcKenzieNZ) [reply readme.md:38]: Please stfu and don't tell me what to do with my resource.

File: readme.md (hunk 7 above)
Comment (TayMcKenzieNZ) [reply readme.md:27]: bla bla bla

**Comment by TayMcKenzieNZ** [comment]
You clearly have no idea what cayo perico looks like originally. i spent 3 years on this map, and you have the audacity to tell me "pls dont reemove the feature list". excuse me? i created that list. you dont do anything but take credit and claim you're "preserving shit"

**Comment by alberttheprince** [comment]
Due to the nature of your responses, I will unfortunately have to reject your pr at this time. Please only submit serious prs intended to improve functionality of this resource.

**Comment by TayMcKenzieNZ** [comment]
You're a joke. Remove my fucking files.

**Comment by Mandalorian531** [comment]
Alberttheprince You're a fraud leave the job to TayMcKenzieNZ, he brought so much to Cayo Perico, unlike you, you're a clown!

**Comment by TayMcKenzieNZ** [comment]
> Alberttheprince You're a fraud leave the job to TayMcKenzieNZ, he brought so much to Cayo Perico, unlike you, you're a clown!


Obviously, I don't own code, and I have no problem with people modifying the code so that Los Santos, Cayo Perico and North Yankton work simultaneously, however in the PR, I simply removed the feature list and my modified files, of which I spent 3+ years on.

He claims he's "preserving the integrity of the resources" however why now? Why suddenly after I've been banned from this shitty toxic community, does our prince suddenly care and why is TheIndra backing it? 🤨🤔 

Very suspicious. I don't believe any preservation is being done here, especially when credits lead to nowhere and don't credit the proper people, and this guy doesn't know codewalker or Lua id his life depended on it.

Happy April Fools, you're a joke.

**Comment by Mandalorian531** [comment]
I also don't understand why he wakes up now for no reason. But what I do know is that his behavior is disgusting. I hope that Karma will take care of him.

**Comment by TayMcKenzieNZ** [comment]
Why even bother 'preserving' resources if you're not going to update them?
Lol I'm going to update RPEmotes and CayoTwoIslands but unfortunately due
to people like this, I have to put it behind a paywall, so if this soo
called Prince wants these updates, he can pay me for it or he can learn to
modify and release his _*own*_ work, without piggy backing off of other
people 🤡

**Comment by Mandalorian531** [comment]
This man is a really bad one, whether he moves his a*s or goes back to sleep with his toxic mentality.

**Comment by TayMcKenzieNZ** [comment]
I see him in the forums policing it and telling people they have to include direct downloads and that X already exists why are you uploading Y, or that something is stolen or leaked; very ironic coming from someone who has expertise in thievery and leaking 🤡

**Comment by BrazyDevelopment** [comment]
Oh dear

**Comment by TayMcKenzieNZ** [comment]
8l4axq

Thanks for letting us know. We agree there is an issue and we're looking into it.

**Comment by alberttheprince** [comment]
> > Alberttheprince You're a fraud leave the job to TayMcKenzieNZ, he brought so much to Cayo Perico, unlike you, you're a clown!
> 
> Obviously, I don't own code, and I have no problem with people modifying the code so that Los Santos, Cayo Perico and North Yankton work simultaneously, however in the PR, I simply removed the feature list and my modified files, of which I spent 3+ years on.
> 
> He claims he's "preserving the integrity of the resources" however why now? Why suddenly after I've been banned from this shitty toxic community, does our prince suddenly care and why is TheIndra backing it? 🤨🤔
> 
> Very suspicious. I don't believe any preservation is being done here, especially when credits lead to nowhere and don't credit the proper people, and this guy doesn't know codewalker or Lua id his life depended on it.
> 
> Happy April Fools, you're a joke.

Happy Apirl Fools day! Although it's March 31st here.

Unfortunately, due to the lack of professionalism, you have displayed here in your replies and insinuations that I should "go back to sleep," I'm going to have to lock this conversation. 

While I appreciate the contributions you've made to this Open source project licensed under GPL 3.0, it seems you have generally lost perspective in this case. 

I hope you can take this as a sign that you should reevaluate your goals, and maybe seek the help and guidance needed to realign yourself with healthy, realistic mindsets on and offline.

**Pull request closed without merging**